pom.xml

```diff
@@ -11,7 +11,7 @@
     <version>1.0.8-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.7.3</jenkins.version>
+        <jenkins.version>2.37</jenkins.version>
```

Comment:
What is the motivation for this version downgrade?

Reply:
Its upgrade from `2.7.3` -> `2.37` (On one version of PR I had `2.150.1` which is latest LTS, but test on it works lot slower, and no one with older version of jenkins will be able to use that plugin)

Reply:
Understood, thanks for clearing that up.